### ISSUE TITLE
Feature Request: Pass signal to onShutdown

### ISSUE THREAD

**Issue by AndrewLeedham** [sebhildebrandt/http-graceful-shutdown#5]
### Issue Type

* [ ] Bug
* [x] Feature Request
* [ ] Question

### Feature
It would be nice if we had access to the signal type that was called in the onShutdown callback.
```javascript
gracefulShutdown(server, {
  signals: 'SIGINT SIGTERM',
  onShutdown: async(signal) => {
    return process.kill(process.pid, signal // = 'SIGINT' || 'SIGTERM');
  }
});
```

### Use case
This would be useful in combination with nodemon, as we could pass the signal to `process.kill` as described here: https://github.com/remy/nodemon#controlling-shutdown-of-your-script

**Comment by sebhildebrandt** [comment]
Nice idea!! Will have a look on it and make a change!

**Comment by sebhildebrandt** [comment]
Just published a new version 2.2.0 that allows to add an input parameter to your shutdown (cleanup) function. Inside your function you can then access the signal type ... see example in the README.md. Hope that this is, what you wanted.

**Comment by AndrewLeedham** [comment]
Wow. That was quick! Thanks @sebhildebrandt 🚀
